### DISCUSSION TITLE
Markdown editor for comments/worknotes.

### DISCUSSION THREAD

**Discussion by CGraabaek** [arnoudkooi/SN-Utils#518] [Ideas]
Would it be possible to add a WYSIWYG editor for comments and worknotes? 
The idea would be to replace the comment field with a markdown editor where it would be easier to format comments for the end users. 

It could utilize the `[code][/code]` to insert the HTML from the WYSIWYG editor. 

I don't know if this is out of the scope of SNUtils, I just thought about this ability.

**Comment by arnoudkooi** [comment]
No, this is outside the scope. You could consider submitting it as an idea on the support portal if you have access to that.
